src/hotspot/share/prims/jvm.cpp

```diff
@@ -4044,3 +4044,22 @@ JVM_ENTRY(void, JVM_VirtualThreadUnmountEnd(JNIEnv* env, jobject vthread, jboole
   fatal("Should only be called with JVMTI enabled");
 #endif
 JVM_END
+
+/*
+ * Return the current class's class file version.  The low order 16 bits of the
+ * the returned jint contains the class's major version.  The high order 16 bits
```

Comment:
typo "the the" across the lines
typo s/contains/contain/

Reply:
Fixed!